config/default.cjs

```diff
@@ -27,6 +28,8 @@ const config = {
     host: SECRET_HOST,
     port: SECRET_PORT,
     secret_key: SECRET_MASTER_KEY,
+    defaultLanguage: SECRET_DEFAULT_LANGUAGE,
```

Comment:
You have to place this in the `__client_config.settings` further down the file, or else it will not be available in the frontend